bindings/python/tests/test_parse.py

```diff
@@ -0,0 +1,30 @@
+# from context import scripts
+import scripts.parse as parse
+
+import clang.cindex as clang
+
+
+def test_node_in_this_file_true():
+    def recursive_call(cursor):
+        for child in cursor.get_children():
+            assert parse.node_in_this_file(child, tu.spelling) == True
+            recursive_call(cursor=child)
+
+    unsaved_files = (("file.cpp", "void function() {}"),)
+    tu = clang.TranslationUnit.from_source(filename="file.cpp", unsaved_files=unsaved_files)
+    recursive_call(cursor=tu.cursor)
+
+
+def test_node_in_this_file_false_none():
+    def recursive_call(cursor):
+        for child in cursor.get_children():
+            if child.location.file:
+                assert parse.node_in_this_file(child, tu.spelling) == False
+            else:
+                assert parse.node_in_this_file(child, tu.spelling) == None
```

Comment:
Doesn't this function only return boolean values?

Reply:
It was returning `None` where `child.location.file` was `None`.
Changed the function in `parse.py`, will change the test too: https://github.com/divmadan/pcl/pull/10/commits/4c5601f17917728e6feacf3c92da4c1c677122cf